1998/0xxx/CVE-1999-0001.json

```diff
@@ -0,0 +1,67 @@
+{
```

Comment:
Expected location (1999\0xxx\CVE-1999-0001.json) of file does not match actual location (1998\0xxx\CVE-1999-0001.json)